cpp/src/sfntly/table/truetype/glyph_table.cc

```diff
@@ -215,10 +215,11 @@ CALLER_ATTACH GlyphTable::Glyph*
 
   ReadableFontDataPtr sliced_data;
   sliced_data.Attach(down_cast<ReadableFontData*>(data->Slice(offset, length)));
-  if (type == GlyphType::kSimple) {
-    glyph = new SimpleGlyph(sliced_data);
-  } else {
-    glyph = new CompositeGlyph(sliced_data);
+  if (sliced_data) {
```

Comment:
|glyph| is a smart pointer, so it starts out in a consistent nullptr state internally, and returns a nullptr on Detach().